readme.markdown

```diff
@@ -148,7 +148,8 @@ Available `opts` options are:
 - opts.timeout = 500. Set a timeout for the test, after which it will fail.
   See test.timeoutAfter.
 - opts.objectPrintDepth = 5. Configure max depth of expected / actual object
-  printing.
+  printing. Environmental variable `NODE_TAPE_OBJECT_PRINT_DEPTH` can set the 
+  desired depth for all tests and overrides any locally-set values.
```

Comment:
should this override? or should the env var merely set the default, so that local intentional overrides still take precedence?

Reply:
Was wondering the same, since good arguments can be made for both approaches. Finally decided to go the current route because:
1. seems more compatible with 12-Factor Apps guidance
2. env vars overriding is how https://www.npmjs.com/package/config works

Reply:
I've no interest in complying with "12-factor apps" by itself; if it isn't the best thing for the project, then it shouldn't be done.

In general, i would agree that an env var should always override an *equivalent setting* - but there is no equivalent for "set every object print depth" in tape. Currently, every one is per-test, and per-test overrides imo should always take precedence.

Reply:
OK, no problem. That makes sense. I was just explaining the logic behind the initial decision, but not doing it that way also makes sense. I will make the change as you requested.